config/routes.rb

```diff
@@ -1,24 +1,58 @@
 OpenStreetMap::Application.routes.draw do
+  # Our api versions are decimals, but controllers cannot start with a number
+  # or contain punctuation
+  def v_string(version)
+    "v#{version.delete('.')}"
```

Comment:
Thinking of semver again, would it be better to replace "." by "_" maybe, so it's `v0_7 ` and `v1_2 `? How about patch version number? Ignore or include?

Reply:
As discussed in the comments, semver would only use major version numbers here. So e.g. `v7` or `v124`. But it's a great point to raise, thank you.